apps/modernization-ui/src/apps/search/patient/result/patientSearchResult.spec.tsx

```diff
@@ -0,0 +1,77 @@
+import { render } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import { PatientSearchResult } from 'generated/graphql/schema';
+import { displayProfileLink, displayPhones, displayEmails } from './patientSearchResult';
+
+jest.mock('name', () => ({
+    displayName: jest.fn(() => jest.fn((name) => `${name.last}, ${name.first}`))
+}));
+jest.mock('address/display', () => ({
+    displayAddress: jest.fn((address) => `${address.street}, ${address.city}`)
+}));
```

Comment:
question:  Why mock these functions to do almost what the actual functions do?  I don't see them being used in the test either